package.json

```diff
@@ -39,9 +39,9 @@
     "prettier:fix": "yarn prettier --write",
     "prettier": "prettier './src/**/*.{js,jsx,ts,tsx}'",
     "release": "electron-builder --mac --linux --windows -p always",
-    "start-mainnet": "REACT_APP_NETWORK=mainnet yarn start",
-    "start-polygon": "REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK=polygon yarn start",
-    "start-bsc": "REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK='bsc' yarn start",
+    "start-mainnet": "yarn start REACT_APP_NETWORK=mainnet",
+    "start-polygon": "yarn start REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK=polygon",
+    "start-bsc": "yarn start REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK='bsc'",
```

Comment:
What is the motivation for this change?

Linux environments don't accept ENV parameters after the command so this won't work

Reply:
I can feel Bill Gates influence in this change

Reply:
Windows machines need them after. Shall I create a `*-win` command for all the Windows users?

Reply:
This won't work on Mac either.

Maybe this'll work universally?

```
export REACT_APP_NETWORK=mainnet; yarn start
```

Reply:
Also, I assume windows users can run a Linux compatibility layer like WSL/Cygwin/MinGW.

Reply:
I'll revert these changes and look into the best option for Windows as it's just me with a Windows machine.

Reply:
> 
> 
> This won't work on Mac either.
> 
> Maybe this'll work universally?
> 
> ```
> export REACT_APP_NETWORK=mainnet; yarn start
> ```

As this didn't work and I'll look into WSL etc.

Reply:
We should use `cross-env` then in this command

Reply:
Try setting them like this: 

```
"start-mainnet": "cross-env REACT_APP_NETWORK=mainnet yarn start",
"start-polygon": "cross-env REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK=polygon yarn start",
"start-bsc": "cross-env REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK='bsc' yarn start",
 ```

Reply:
I think using WSL is a better option (for us) because I know some commands in other gnosis' packages heavily depend on the shell (scripts, etc.)

Richard knows how to set up it properly, and you can ask him

Reply:
I've added `cross-env` for the time being.